onnxmltools/convert/sklearn/operator_converters/KNN.py

```diff
@@ -24,6 +32,7 @@ def convert_sklearn_knn(scope, operator, container):
     # C: Number of classes
     # input: test set input
     # output: test set output 
+    # output_prob: test set class probabilities 
```

Comment:
You need to indicate when the sub-grap for probability calculation would be generated.